osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -317,6 +354,30 @@ private void load(OverlayColourProvider colourProvider)
             };
         }
 
+        private void deleteComment()
+        {
+            dialogOverlay.Push(new ConfirmDialog("Do you really want to delete your comment?", () =>
+            {
+                actionsContainer.Hide();
+                actionsLoading.Show();
+                var request = new CommentDeleteRequest(Comment.Id);
+                request.Success += _ =>
+                {
+                    actionsLoading.Hide();
+                    AutoSizeAxes = Axes.None;
+                    Masking = true;
+                    this.ResizeHeightTo(0, 1000, Easing.Out);
+                    this.FadeOut(1000, Easing.Out).Expire();
```

Comment:
I feel this is probably a bit over-the-top? I'd choose either the resize or the fade (one or the other) and change the duration to ~400-500ms.